opensrp-eusm/src/main/java/org/smartregister/eusm/repository/AppStructureRepository.java

```diff
@@ -156,6 +157,11 @@ public int countOfStructures(String nameFilter, String locationParentId) {
 
 
         Location location = EusmApplication.getInstance().getUserLocation();
+        if (location == null) {
+            location = new Location("temp");
+            location.setLongitude(0.000);
+            location.setLatitude(0.000);
+        }
```

Comment:
why is this needed>

Reply:
Seen you are calculating distances on sqlite. Its not possible to calculate accurate distances on sqlite since it lacks some cos/sin/tan functions

Reply:
I use that as the default location, ideally the app should work without it.

Reply:
I use it a default location but when the app is being used it will be replaced by the actual location.